src/core/system.c

```diff
@@ -105,20 +110,46 @@ void Sys_ResetTempMem()
 SYS TIME
 ===============================================================================
 */
-float Sys_FloatTime (void)
+
+#if _MSC_VER
+typedef struct timeval {
+	long tv_sec;
+	long tv_usec;
+} timeval;
+
+int gettimeofday(struct timeval * tp, struct timezone * tzp)
```

Comment:
there are: 
 - SDL_GetTicks() // bad precision: ms - do not use;
 - SDL_GetPerformanceCounter() and SDL_GetPerformanceFrequency() // is SDL_GetPerformanceFrequency() slow, can I use it once, on app init?